matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/rendezvous/Rendezvous.kt

```diff
@@ -225,12 +221,7 @@ class Rendezvous(
             // request secrets from the verifying device
             Timber.tag(TAG).i("Requesting secrets from $verifyingDeviceId")
 
-            session.sharedSecretStorageService().let {
-                it.requestSecret(MASTER_KEY_SSSS_NAME, verifyingDeviceId)
-                it.requestSecret(SELF_SIGNING_KEY_SSSS_NAME, verifyingDeviceId)
-                it.requestSecret(USER_SIGNING_KEY_SSSS_NAME, verifyingDeviceId)
-                it.requestSecret(KEYBACKUP_SECRET_SSSS_NAME, verifyingDeviceId)
-            }
+            session.sharedSecretStorageService().requestMissingSecrets()
```

Comment:
I understand that we request the secrets from all our devices, and not the device which was used to verify (`verifyingDeviceId`). I think it's fine, but maybe update the comment and the log above?

Reply:
updated